pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeOverview/tooltips.tsx

```diff
@@ -85,6 +85,47 @@ export const ValueBytesTooltip: React.FC<
   </Tooltip>
 );
 
+export const RangeKeyBytesTooltip: React.FC<
+  TooltipProps & {
+  nodeName: string;
+}
+  > = props => (
+  <Tooltip
+    {...props}
+    placement="bottom"
+    title={
+      <div className="tooltip__table--title">
+        <p>
+          Number of bytes stored in range keys on node {props.nodeName || "NodeName"}.
+        </p>
+      </div>
+    }
+  >
+    {props.children}
+  </Tooltip>
+);
+
+export const RangeValueBytesTooltip: React.FC<
+  TooltipProps & {
+  nodeName: string;
+}
+  > = props => (
+  <Tooltip
+    {...props}
+    placement="bottom"
+    title={
+      <div className="tooltip__table--title">
+        <p>
+          Number of bytes stored in range values on node{" "}
```

Comment:
nit: we should use "MVCC range key values" (and similarly "MVCC range keys" above), to disambiguate them. "range keys" or "range values" can sound like e.g. "keys in the Raft range", which is a different thing.

Reply:
Thanks!